internal/image/validate_test.go

```diff
@@ -116,11 +114,8 @@ func TestValidateImage(t *testing.T) {
 		t.Run(c.name, func(t *testing.T) {
 			fs := afero.NewMemMapFs()
 
-			p := &policy.Policy{
-				EnterpriseContractPolicySpec: v1alpha1.EnterpriseContractPolicySpec{},
-				CheckOpts:                    &cosign.CheckOpts{},
-				EffectiveTime:                time.Now().UTC(),
-			}
+			p, err := policy.NewOfflinePolicy(context.TODO(), "")
```

Comment:
There's a context a couple of lines below. Maybe do this after and use that context?